packages/suite/src/components/wallet/WalletLayout/AccountsMenu/TokensCount.tsx

```diff
@@ -1,14 +1,11 @@
 import { MouseEventHandler } from 'react';
 import styled from 'styled-components';
-import { Button } from '@trezor/components';
-import { spacingsPx } from '@trezor/theme';
+import { Badge } from '@trezor/components';
 
-const ButtonBadge = styled(Button)`
-    margin-left: ${spacingsPx.xs};
-    background: ${({ theme }) => theme.backgroundSurfaceElevation2};
-    color: ${({ theme }) => theme.textSubdued};
-    font-size: 10px;
-    padding: 1px 8px;
+const Container = styled.div`
```

Comment:
how about `styled(Badge)` instead? Also, it should be round. 🙏

Reply:
Overring obviously should not happen! You are right, dynamically changing border-radius when Badge has 1 character is cool as hell. And if you can create an issue or add it here it will definitely be nice. If not here, then a temporary override is better IMO. And sometimes you simply need to change margins or something, so `styled(Something)` is not always terrible.

Reply:
It's not terrible, but it adds another complexity in my opinion. You never know if you are not overriding something inside of the component and searchability for refactoring is harder. Anyway I tried to use overriding here and it's not that easy in this case, so let's keep Container.

<img width="270" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/daa2ce6d-961c-4088-8274-3a7158893684">

<img width="723" alt="image" src="https://github.com/trezor/trezor-suite/assets/858321/63a8a259-5790-42ba-bd17-78831ab557bb">

Reply:
I guess that's because `Badge` did not have a `className` prop? 🤔  Fixed 8ea1554da6

![image](https://github.com/trezor/trezor-suite/assets/45338719/7e65e6f8-4a2d-4b4d-883a-376de0889cfb)
![image](https://github.com/trezor/trezor-suite/assets/45338719/61fcc8aa-d4d4-4a33-a44e-df0b5cb9ea67)